frontend/sw360-portlet/src/main/webapp/html/components/edit.jsp

```diff
@@ -121,6 +128,10 @@
 </core_rt:if>
 
 <script>
+    var permissions = {  'write':  stringToBoolean("${permissions.makePermission(component, currentUser).isActionAllowed(RequestedAction.DELETE)}"),
+                         'delete': stringToBoolean("${permissions.makePermission(component, currentUser).isActionAllowed(RequestedAction.WRITE)}")
```

Comment:
isn't it the case that you convert a boolean to string here by enclosing it in quotes and then send it to `stringToBoolean()` to convert back to boolean? Can this be avoided?